app/components/Header/DropdownContents/EventsDropdown.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import React from 'react';
+import Icon from '../../Icon';
+import styles from '../Header.css';
+
+const EventsDropdown = () => {
+  return (
+    <div className={styles.eventsDropdownEl}>
+      <div className={styles.dropdownSection} data-first-dropdown-section>
+        <a className={styles.dropdownLink} href="/events">
+          <div className={styles.iconFrame}>
+            <Icon name="list" size={25} />
+          </div>
+          <div className={styles.textAndArrow}>
+            <h4>Kommende arrangementer</h4>
+            <Icon className={styles.icon} name="arrow-forward" size={20} />
+          </div>
+        </a>
+        <a className={styles.dropdownLink} href="/events/calendar">
+          <div className={styles.iconFrame}>
+            <Icon name="calendar" size={25} />
+          </div>
+          <div className={styles.textAndArrow}>
+            <h4>Kalender</h4>
+            <Icon className={styles.icon} name="arrow-forward" size={20} />
+          </div>
+        </a>
+      </div>
+      <div className={styles.dropdownSection}>
+        <a className={styles.altText} href="/users/me">
+          Dine arrangementer
+        </a>
+      </div>
```

Comment:
There are a bunch of `<a>` tags here. That will wreck the SPA. Use `<Link>` from react-router

Reply:
This is also quite experimental, but ye, totally agree!

Reply:
This was also experimental, but agree! 😄